website/views/options/settings.jade

```diff
@@ -379,12 +379,12 @@ script(id='partials/options.settings.subscription.html',type='text/ng-template')
 
           .form-inline
             .form-group
-              input.form-control(type='text', ng-model='_subscription.coupon', placeholder='Promo Code')
+              input.form-control(type='text', ng-model='_subscription.coupon', placeholder= env.t('couponPlaceholder'))
```

Comment:
I used coupon since it used ng-model='_subscription.coupon' and I couldn't find a promo code translation in the en folder.